### PR TITLE
add www alias for sdiy.info

### PR DESCRIPTION
**Pull request by robkam**

### CODE REVIEW

**Comment by Reception123** [comment]
@robkam www goes under "Services" not "Wildcard"

**Comment by robkam** [comment]
It appears not to work on those wikis with www in services but it does for where it's in wildcard.

**Comment by Reception123** [comment]
No, it does not depend on that. It depends on whether the WWW domain has a certificate or not. Please add www under services.

File: zones/sdiy.info [hunk 1]
```diff
@@ -11,6 +11,7 @@ $ORIGIN sdiy.info.
 
 ; Wildcard services
 @		DYNA	geoip!mw
+*		DYNA	geoip!mw
```
Comment (Reception123) [inline zones/sdiy.info]: There's no point to leave this here

**Pull request merged**